Documentation/Upgrade/rook-upgrade.md

```diff
@@ -78,7 +78,7 @@ section for instructions on how to change the default namespaces in `common.yaml
 Then apply the latest changes from v1.11 and update the Rook Operator image.
 
 ```console
-kubectl apply -f common.yaml -f crds.yaml
+kubectl apply --server-side -f common.yaml -f crds.yaml
```

Comment:
--force-conflicts was not required

Reply:
Good catch. I used `replace` in the upgrade test but forgot to change it here. I'll make a small PR.